dxDiagram: Update API descriptions (#1384)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDiagram/1 Configuration/customShapes/allowEditImage.md

---
---
##### shortDescription
Specifies whether a card shape's image can be edited.

---
This option is in effect for shapes with the [baseType](/api-reference/10%20UI%20Widgets/dxDiagram/1%20Configuration/customShapes/baseType.md '/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/customShapes/#baseType') option set to `"cardWithImageOnLeft"`, `"cardWithImageOnTop"`, or `"cardWithImageOnRight"`.

> A card shape is designed to contain an image. The [defaultImageUrl](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/customShapes/#defaultImageUrl) option specifies an image that is displayed in the card by default. Set the **allowEditImage** option to `true` to allow users to change the image via the context menu commands.

Reviewer: We need to revise this so that we don't use "by default" or "via". Can we say the following instead?

```suggestion
A card shape is designed to contain an image. The [defaultImageUrl](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/customShapes/#defaultImageUrl) option specifies the default image that is displayed in the card. Set the **allowEditImage** option to `true` to allow users to change the image from a context menu.
```
